l10n/en/products/vpn/platforms/desktop.ftl

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/
+
+# Desktop strings
+vpn-desktop-hero-headline = Secure your internet connection with { -brand-name-mozilla-vpn } for desktop
+vpn-desktop-hero-copy = Get fast, easy security on your desktop and laptop computer through { -brand-name-mozilla }, one of the most trusted names in technology.
+
+vpn-desktop-ability = { -brand-name-mozilla-vpn } gives you the ability to connect confidently and keep your information safe from prying eyes and data thieves when you’re online. { -brand-name-mozilla-vpn } uses advanced { -brand-name-mozilla-vpn-wireguard }® protocol to encrypt personal data on your laptop or desktop computer — for up to five devices. Stream shows, play games, shop, and go about your daily life online knowing that you’re secure.
+
+vpn-desktop-maker-of-firefox = As the maker of the { -brand-name-firefox } web browser and one of the most trusted fighters for internet privacy, { -brand-name-mozilla } is committed to ensuring you feel safe and respected every time you go online.
+
+vpn-desktop-wifi-headline = Feel safe using public wifi
+vpn-desktop-wifi-copy = Our VPN shields your internet connection from hackers and spies, so you can shop from the coffee shop, check your bank account from the train and go about your online business anywhere without safety concerns.
+
+vpn-desktop-about-headline = About { -brand-name-mozilla }
+vpn-desktop-about-copy = Best known for our { -brand-name-firefox-browser }, { -brand-name-mozilla } is a global community of users, contributors and developers dedicated to keeping the power of the internet in people's hands. Founded in 1998, we’re one of the pioneers of the free and open web — and it’s our mission to keep it that way.
+
+vpn-desktop-privacy-headline = Privacy in one click
+vpn-desktop-privacy-copy = The { -brand-name-mozilla-vpn } app encrypts your internet connection and hides the IP address on your mobile phone or tablet using { -brand-name-mozilla-vpn-wireguard }® advanced protocol.
+
+vpn-desktop-servers-headline = Connect to servers all over the world
+vpn-desktop-servers-copy = Browse from Brazil. Game from Japan. Stream from Mexico. { -brand-name-mozilla-vpn } lets you change your phone or computer’s location to one of { -vpn-features-servers }.
+
+vpn-desktop-devices-headline = Protect up to { -vpn-available-devices } devices
```

Comment:
Nit: the device numbers her should be handled using variables instead of placeholders (see examples in platforms/shared.ftl)

Reply:
Wouldn't this create more loose ends? Specifically, in every `*.html` file that references this FTL file will have their own instance of the number. Is there a different way to only set this number in one location?

Reply:
We should try and keep things like constants and variables out of translation files, as then they end up in Pontoon and out of our control (as localizers are free to change them). If you want a canonical source for this data then I'd suggest passing a context variable in the view to these templates (like we do currently for the price). Happy to help there if you need it 👍